harness-ci-factory/main.tf

```diff
@@ -0,0 +1,128 @@
+module "organization" {
+  source = "github.com/harness-community/terraform-harness-structure//modules/organizations"
```

Comment:
We should reference these via the registry source instead of git https://github.com/harness-community/solutions-architecture/blob/enhancement/update-module-source/terraform-development-factory/main.tf#L2-L3.